### ISSUE TITLE
Disabling shell in subprocess call requires arguments as list

### ISSUE THREAD

**Issue by Atesca** [L2jLiga/grunt-webfonts#35]
This commit https://github.com/L2jLiga/grunt-webfonts/commit/0fbd5ee30bda79b1c7fc8123a324f73d8e6f926e broke the processing of fonts for me.

According to the python subprocess documentation, if parameters are to be passed to the executable they have to be passed as a list, otherwise if they are passed in the string they are interpreted to be the commant to be executed.

I think it should look like
```
import shlex
...
call(shlex.split('ttfautohint --symbol --fallback-script=latn --no-info "%(font)s.ttf" "%(font)s-hinted.ttf" && mv "%(font)s-hinted.ttf" "%(font)s.ttf"' % {'font': fontfile}))
...
call(shlex.split('python "%(path)s/../../bin/eotlitetool.py" "%(font)s.ttf" --output "%(font)s.eot"' % {'path': scriptPath, 'font': fontfile}))
```

**Comment by L2jLiga** [comment]
Hi there, would you like to send PR to fix the issue?

**Comment by Atesca** [comment]
@L2jLiga done

**Comment by L2jLiga** [comment]
Thanks! Released as v4.0.3
